test/web-auth/cross-origin-authentication.test.js

```diff
@@ -166,7 +166,9 @@ describe('auth0.WebAuth.crossOriginAuthentication', function() {
               error_description: 'a huge error string'
             },
             code: 'any error',
-            description: 'a huge error string'
+            description: 'a huge error string',
+            error: 'any error',

```

Comment:
This should probably be a machine_name without spaces

Reply:
doesn't really matter for the tests though..